tests/specal_layer_width_test.py

```diff
@@ -40,5 +40,5 @@ def test_substitution_layer_width():
     masters, instances = load_to_ufos(glyphs_file_path(), include_instances=True)
     assert masters[0]["B"].width == 500
     assert masters[1]["B"].width == 600
-    assert masters[0]["B.BRACKET.40"].width == 510
-    assert masters[1]["B.BRACKET.40"].width == 610
+    assert masters[0]["B.BRACKET.wght_40_100"].width == 510
+    assert masters[1]["B.BRACKET.wght_40_100"].width == 610
```

Comment:
I know this file pre-existed this PR, but it would make sense to fix the typo in the test filename => `git mv tests/spec{,i}al_layer_width_test.py`